tokens/src/components/Pagination.json

```diff
@@ -1,99 +1,101 @@
 {
-  "pagination": {
-    "padding": {
-      "y": {
-        "base": { "value": ".625rem", "source": "$pagination-padding-y" },
-        "sm": { "value": ".8rem", "source": "$pagination-padding-y-sm" },
-        "lg": { "value": ".75rem", "source": "$pagination-padding-y-lg" }
-      },
-      "x": {
-        "base": { "value": "1rem", "source": "$pagination-padding-x" },
-        "sm": { "value": ".6rem", "source": "$pagination-padding-x-sm" },
-        "lg": { "value": "1.5rem", "source": "$pagination-padding-x-lg" }
-      },
-      "icon": { "value": ".5rem", "source": "$pagination-padding-icon" }
-    },
-    "margin": {
-      "x": { "value": ".5rem", "source": "$pagination-margin-x" },
-      "y": { "value": ".5rem", "source": "$pagination-margin-y" }
-    },
-    "line": {
-      "height": {
-        "value": "1.5rem", "source": "$pagination-line-height"
-      }
-    },
-    "font": {
-      "size": {
-        "sm": { "value": ".875rem", "source": "$pagination-font-size-sm" }
-      }
-    },
-    "icon": {
-      "size": {
-        "base": { "value": "1.375rem", "source": "$pagination-icon-size" },
-        "sm": { "value": "1rem", "source": "$pagination-icon-size-sm" }
-      },
-      "width": { "value": "2.25rem", "source": "$pagination-icon-width" },
-      "height": { "value": "2.25rem", "source": "$pagination-icon-height" }
-    },
-    "toggle": {
-      "border": {
-        "base": { "value": ".3125rem", "source": "$pagination-toggle-border" },
-        "sm": { "value": ".25rem", "source": "$pagination-toggle-border-sm" }
-      }
-    },
-    "secondary": {
-      "height": {
-        "base": { "value": "2.75rem", "source": "$pagination-secondary-height" },
-        "sm": { "value": "2.25rem", "source": "$pagination-secondary-height-sm" }
-      }
-    },
-    "color": {
+  "color": {
+    "pagination": {
       "base": { "value": "{color.link.value}", "source": "$pagination-color" },
```

Comment:
should probably create new key `text` for these variants

Reply:
done